README.md

```diff
@@ -95,4 +115,11 @@ See the full list in https://plone.github.io/pastanaga-angular/@@palette.
 Then, some components have a second layer of tokens, for example buttons have a list of token for each aspect and state (*e.g.* `$color-text-button-primary-solid`, `$color-background-button-primary-solid`,…).
 So you can have your own theme by overwriting the whole color palette or just by changing some aspects of some components.  
 
-
+In angular.json configuration you should add `src` folder to be included in style preprocessing:
```

Comment:
In angular.json configuration, you should declare the folder path containing your `pastanaga-core-overrides.scss` and `pastanaga-component-overrides.scss` files in style preprocessing options. For example, if these two files are in `src`: